server/cdk/test/cdk.test.ts

```diff
@@ -0,0 +1,17 @@
+// import * as cdk from 'aws-cdk-lib';
+// import { Template } from 'aws-cdk-lib/assertions';
+// import * as Cdk from '../lib/cdk-stack';
+
+// example test. To run these tests, uncomment this file along with the
+// example resource in lib/cdk-stack.ts
+test('SQS Queue Created', () => {
+//   const app = new cdk.App();
```

Comment:
Why is this all commented out? lol

Reply:
That may or may not be the test that came with the starter project. I could update it to be snapshot tests for stability, but that might be annoying during development